ksql-workshop/ksql-workshop.sql

```diff
@@ -0,0 +1,32 @@
+CREATE STREAM CUSTOMERS_SRC (id BIGINT, first_name VARCHAR, last_name VARCHAR, email VARCHAR, gender VARCHAR, club_status VARCHAR, comments VARCHAR) WITH (KAFKA_TOPIC='customers', VALUE_FORMAT='JSON');
```

Comment:
They're pretty much the same, except the `.sql` is what I'd get workshop participants to follow, and because of the Avro/explicit column naming problem, I wouldn't want workshop participants to use that version. 
Not sure what to suggest here, unless we hide one of them in a subfolder?

Reply:
Can you just add comments to ksql.commands for the ones that differ?

Reply:
I'd rather not, since it's less usable for workshop participants. Given that https://github.com/confluentinc/ksql/issues/1530 sounds like a way off being fixed, I think we either need to accept two versions of this file here, or hide one in a sub folder (my suggestion would be put `ksql.commands` in `/scripts` since it's only called from a shell script, rather than the user (workshop participant) manually locating it

Reply:
There are reasons to not move `ksql.commands` to `/scripts` (consistency with other demos).

Alternate suggestion:
1. Leave file names as they are
2. Add a comment `--` to the top of `ksql.commands` to explain its purpose (run automatically as part of `start.sh`)
3. Add a comment `--` to the top of `ksql-workshop.sql` to explain its purpose (run manually as part of the workshop, referenced from the `adoc`)
4. Add a line in the `ksql-workshop.adoc` file itself to point users to that file.  Cross-referencing is good for greppers

Reply:
Good plan. I've updated the files as suggested.